frontend/src/game/game.jsx

```diff
@@ -61,10 +61,6 @@ class Game extends Component {
 
     componentWillUnmount() {
         clearInterval(this.interval);
-        this.setState({
-                valid_placements: []
-            }
-        );
```

Comment:
Ignore all else automatic style formatting (my vscode does that).
This is the only change which matter.
Without this change you see a warning (in the chrome developer tools) about not allowed to change state in `componentWillUnmount` because will result with mem leak.
Not sure what will the affect of that be. I don't see it as needed because `onDrop` you're doing that too.
@amirkeren wdyt? Will it cause a bug? Do you remember why did you add it here?

Reply:
Reverted (in master). Saw a bug in the game without it. it's just a warning so not critical to invest in a workaround.